app/controllers/RelayTour.scala

```diff
@@ -99,6 +99,11 @@ final class RelayTour(env: Env, apiC: => Api, prismicC: => Prismic) extends Lila
       env.relay.api.deleteTourIfOwner(tour) inject Redirect(routes.RelayTour.by(me.username)).flashSuccess
   }
 
+  def subscribe(id: TourModel.Id, isSubscribed: Boolean) = Auth { _ ?=> me ?=>
+    env.relay.api.subscribe(id, me.userId, isSubscribed)
+    jsonOkResult
```

Comment:
```scala
env.relay.api.subscribe(id, me.userId, isSubscribed) inject jsonOkResult
```
so that the ok response is only sent when, and if, the subscription is done successfully